buildlib/io_demo/io-demo.yml

```diff
@@ -36,10 +36,20 @@ jobs:
           ./contrib/configure-release --prefix=$(Build.Repository.LocalPath)/install
           make -j`nproc`
           make install
-          echo "##vso[task.setvariable variable=workspace;isOutput=true]$(Build.Repository.LocalPath)/buildlib"
-          echo "##vso[task.setvariable variable=io_demo_exe;isOutput=true]$(Build.Repository.LocalPath)/install/bin/io_demo"
         displayName: Build
         name: build
+      - task: CopyFiles@2
+        inputs:
+          sourceFolder: '$(Build.Repository.LocalPath)'
+          contents: |
+            buildlib/**
```

Comment:
is it really needed?

Reply:
Yes, place with currapter , helper and etc scripts for work, can copy only needed script